packages/content-management/content_editor/src/services.tsx

```diff
@@ -126,9 +125,9 @@ export const ContentEditorKibanaProvider: FC<ContentEditorKibanaDependencies> =
 
   const openFlyout = useCallback(
     (node: ReactNode, options: OverlayFlyoutOpenOptions) => {
-      return coreOpenFlyout(toMountPoint(node), options);
+      return coreOpenFlyout(toMountPoint(node, { theme$: core.theme.theme$ }), options);
     },
-    [toMountPoint, coreOpenFlyout]
+    [coreOpenFlyout, toMountPoint, core]
```

Comment:
nit: can we deconstruct `theme$` and only add it as dependency instead of the full `core` object?